src/azure-cli/azure/cli/command_modules/synapse/manual/operations/integrationruntime.py

```diff
@@ -46,10 +46,11 @@ def Managed_Create(cmd, client, resource_group_name, workspace_name, integration
 
 
 def Selfhosted_Create(cmd, client, resource_group_name, workspace_name, integration_runtime_name,
-                      description=None, if_match=None, no_wait=False):
+                      description=None, if_match=None, no_wait=False, enableselfcontainedia=None):
```

Comment:
```suggestion
                      description=None, if_match=None, no_wait=False, enable_self_contained_ia=None):
```
Better to follow python variable naming rule